src/models/KeyboardModel.ts

```diff
@@ -222,9 +222,15 @@ export default class KeyboardModel {
         if (typeof item === 'string') {
           keymapItem = new KeymapItem(curr, item);
         } else {
-          const op = item as KeyOp;
-          const label = keyRow[++col] as string; // next item should be string(row,col)
+          let op = item as KeyOp;
+          let nextOp = keyRow[++col];
+          while (typeof nextOp != 'string') {
+            // in case of continuing KeyOp
+            op = Object.assign({}, op, nextOp);
+            nextOp = keyRow[++col];
+          }
```

Comment:
@adamrocker Don't we need to concern that the loop is not finished because the type of `nextOp` never be string?

Reply:
ah..., it's definitely meaningless KEL but might happen. I'm gonna fix it. Thanks!

Reply:
and this https://github.com/remap-keys/remap/pull/393/commits/2827966ff43ca81002197135ccbb8dae3b4a802e